src/analysis/retail/warrior/arms/modules/talents/Skullsplitter.tsx

```diff
@@ -16,25 +24,62 @@ class Skullsplitter extends Analyzer {
 
   protected abilityTracker!: AbilityTracker;
 
+  private usesTideOfBlood = false;
+
+  private totalDotDamageDone = 0;
+
   constructor(options: Options) {
     super(options);
     this.active = this.selectedCombatant.hasTalent(TALENTS.SKULLSPLITTER_TALENT);
+    if (!this.active) {
+      return;
+    }
+    this.usesTideOfBlood = this.selectedCombatant.hasTalent(TALENTS.TIDE_OF_BLOOD_TALENT);
+    this.addEventListener(
+      Events.damage.by(SELECTED_PLAYER).spell(TALENTS.SKULLSPLITTER_TALENT),
+      this.onSkullSplitterDamage,
+    );
+  }
+
+  private sumUpDamage(events: DamageEvent[]): number {
+    if (events.length === 0) {
+      return 0;
+    }
+    let sum = 0;
+    events.forEach((event) => (sum += event.amount + (event.absorb || 0)));
```

Comment:
This could also be done via `reduce`:
```suggestion
    return events.reduce((acc, event) => acc + event.amount + (event.absorb ?? 0), 0);
```

Reply:
oh thats neat, havent really looked into all the ts/js stuff a lot yet, good to know